Packs/PrismaCloudCompute/Scripts/PCComputeImageComplianceIssuesButton/PCComputeImageComplianceIssuesButton.yml

```diff
@@ -0,0 +1,45 @@
+commonfields:
+  id: PCComputeImageComplianceIssuesButton
+  version: -1
+name: PCComputeImageComplianceIssuesButton
+script: ''
+type: python
+tags: []
+comment: "This script runs the \"prisma-cloud-compute-images-scan-list\" command for a specific container id and returns details about its compliance issues, if found. If any compliance issues found, it will create a new tab in the layout called \"Detailed Compliance Issues\" showing the issues details.\nReturns the following fields for each compliance ID:\n- Compliance ID\n- Cause\n- Severity\n- Title\n- Description."
```

Comment:
```suggestion
comment: "This script runs the \"prisma-cloud-compute-images-scan-list\" command for a specific container ID and returns details about its compliance issues, if found. If any compliance issues found, it will create a new tab in the layout called \"Detailed Compliance Issues\" showing the issues details.\nReturns the following fields for each compliance ID:\n- Compliance ID\n- Cause\n- Severity\n- Title\n- Description."
```